src/utils/colorGen.ts

```diff
@@ -33,9 +33,9 @@ export function genRGBColor(r, g, b) {
   g = g.toString(16);
   b = b.toString(16);
 
-  if (r.length === 1) r = "0" + r;
-  if (g.length === 1) g = "0" + g;
-  if (b.length === 1) b = "0" + b;
+  if (r.length === 1) r = `0${r}`;
+  if (g.length === 1) g = `0${g}`;
+  if (b.length === 1) b = `0${b}`;
```

Comment:
*clicks* Noice (you could put it on one line but don't get me started about that, it'll be absolute spaghetti hell 🤣 )